content/en/blog/falco-plugin-k8saudit-eks.md

```diff
@@ -51,68 +56,314 @@ With:
+Wed May 10 14:07:18 2023: Loading plugin 'k8saudit-eks' from file /usr/share/plugins/libk8saudit-eks.so                                                                                                
+Wed May 10 14:07:18 2023: Loading plugin 'json' from file /usr/share/plugins/libjson.so                                                                                                                
+Wed May 10 14:07:18 2023: Loading rules from file /etc/k8s_audit_rules.yaml                                                                                                                            
+Wed May 10 14:07:18 2023: Loading rules from file /etc/rules.d/override-k8saudit.yaml                                                                                                                  
+Wed May 10 14:07:18 2023: Starting health webserver with threadiness 4, listening on port 8765                                                                                                               
+Wed May 10 14:07:18 2023: Enabled event sources: k8s_audit                                                                                                                                                   
+Wed May 10 14:07:18 2023: Opening capture with plugin 'k8saudit-eks' 
 ```
 
+And get alers:
 ```shell
-16:07:42.045023000: Warning user=eks:certificate-controller verb=get target=eks-certificates-controller target.namespace=kube-system resource=configmapsEvents detected: 1
-Rule counts by severity:
-   WARNING: 1
-Triggered rules by rule name:
-   Dummy rule: 1
-Syscall event drop monitoring:
-   - event drop detected: 0 occurrences
-   - num times actions taken: 0
+14:09:17.699479000: Informational K8s Serviceaccount Created (user=system:node:ip-192-168-34-135.ec2.internal serviceaccount=default ns=default resource=serviceaccounts resp=201 decision=allow reason=)
+14:09:24.046826000: Notice Attach/Exec to pod (user=kubernetes-admin pod=kubecon resource=pods ns=default action=exec command=sh)
 ```
 
-The referencies to `eks` proves it works!!!
+### Kubernetes installation
+
+Let's see now how to install the `k8saudit-eks` plugin with `falcoctl` tool. We'll use the official Helm chart with this `values.yaml` file:
+```yaml
+serviceAccount:
+  create: true
+  annotations:
+    - eks.amazonaws.com/role-arn: {ROLE_ARN}
+
+falco:
+  rules_file:
+    - /etc/falco/k8s_audit_rules.yaml
+    - /etc/falco/rules.d
+  plugins:
+    - name: k8saudit-eks
+      library_path: libk8saudit-eks.so
+      init_config:
+        shift: 10
+        polling_interval: 10
+        use_async: false
+        buffer_size: 500
+      open_params: "{CLUSTER_NAME}"
+    - name: json
+      library_path: libjson.so
+      init_config: ""
+  load_plugins: [k8saudit-eks, json]
+
+falcosidekick:
+  enabled: true
+
+driver:
+  enabled: false
```

Comment:
Now I see that you have disabled it.

Reply:
You were right, I forgot to mention in the local installation that I did it. I fixed.